rpmio/rpmstrpool.cc

```diff
@@ -213,8 +213,8 @@ static void poolHashPrintStats(rpmstrPool pool)
             }
         }
     }
-    fprintf(stderr, "Hashsize: %i\n", ht->numBuckets);
-    fprintf(stderr, "Keys: %i\n", ht->keyCount);
+    fprintf(stderr, "Hashsize: %li\n", ht->numBuckets);
+    fprintf(stderr, "Keys: %li\n", ht->keyCount);
```

Comment:
This happens to compile on x86_64 but isn't right: size_t isn't guaranteed long, and "i" denotes signed integer where this is changing them to unsigned. %zu is the appropriate formatting for size_t.